addons/website_forum/controllers/website_forum.py

```diff
@@ -154,7 +165,12 @@ def questions(self, forum, tag=None, page=1, filters='all', my=None, sorting=Non
             'sorting': sorting,
             'search': fuzzy_search_term or search,
             'original_search': fuzzy_search_term and search,
+            'author': author,
```

Comment:
Would you mind putting author on the top of the list ? (At some point we will reorder them all, just not now to not mess with rebases)